cardano-explorer-db/src/Explorer/DB/Types.hs

```diff
@@ -10,14 +11,28 @@ module Explorer.DB.Types
   ) where
 
 
-import           Data.Fixed (Micro)
+import           Data.Aeson.Encoding       (unsafeToEncoding)
+import           Data.Aeson.Types          (ToJSON (toEncoding, toJSON))
+import qualified Data.ByteString.Builder   as BS (string8)
+import           Data.Fixed (Micro, showFixed)
 import           Data.Text (Text)
 import qualified Data.Text as Text
 import           Data.Word (Word64)
+import           GHC.Generics              (Generic)
 
 newtype Ada = Ada
   { unAda :: Micro
-  } deriving (Eq, Ord, Show)
+  } deriving (Eq, Ord, Generic)
+
+instance ToJSON Ada where
+    -- https://github.com/bos/aeson/issues/227#issuecomment-245400284
```

Comment:
Why is that comment there? It points to an issue from 2014. There have been multiple Aeson major versions released since then.

Reply:
Ah so that comment came from the old code. We should delete it, but make sure we are using the currently accepted way of doing that encodng.